spec/system/edit_submission_spec.rb

```diff
@@ -126,22 +126,28 @@
       within('.btn-group-toggle') do
         find('label', text: 'Yes').click
       end
-      attach_file 'Upload supplemental files', file_fixture('supplémental_1.pdf')
 
-      within('#supplemental-files-table') do
-        expect(page).to have_css('th', text: 'supplémental_1.pdf')
-        click_link_or_button 'Remove'
+      # Creating a temporary copy of a supplemental file with a diacritic.
+      Dir.mktmpdir do |temp_dir|
+        supplemental_file = File.join(temp_dir, 'supplémental_1.pdf')
+        FileUtils.cp(file_fixture('supplemental_1.pdf'), supplemental_file)
+        attach_file 'Upload supplemental files', supplemental_file
+
+        within('#supplemental-files-table') do
+          expect(page).to have_css('th', text: 'supplémental_1.pdf')
+          click_link_or_button 'Remove'
+        end
+
+        within('.btn-group-toggle') do
+          find('label', text: 'Yes').click
+        end
+
+        attach_file 'Upload supplemental files', [
+          supplemental_file,
+          file_fixture('supplemental_2.pdf')
+        ]
```

Comment:
The most important thing to test with files containing diacritics is the signature page generation, I believe. Does this test generate a signature page with a supplemental file named with diacritics?

Reply:
It looks like the test for `SignaturePageService` invokes the service with a submission that includes a supplemental file with a diacritic.

Reply:
https://github.com/sul-dlss/heracles-etd/blob/main/spec/services/signature_page_service_spec.rb#L8
which uses https://github.com/sul-dlss/heracles-etd/blob/main/spec/factories/supplemental_files.rb#L10